schema/patient.js

```diff
@@ -67,7 +67,7 @@ const patientSchema = `
     gender: Gender,
     age: Int,
     runNumber: Int,
-    barcodeValue: String,
+    barcodeValue: String!,
```

Comment:
i noticed that the `addPatient` mutation has `triageLevel` as nullable (line 37) - we should have the required fields for that and this `type Patient` to match (i.e. `barcodeValue`, `collectionPoint`, `triageLevel` should be required/non-nullable for both the type and the `addPatient` mutation

(not exactly in the scope of your ticket but i figure it's a quick minor change that's easy to make here)